executor/src/metrics.rs

```diff
@@ -19,12 +28,30 @@ impl ExecutorMetrics {
                 registry
             )
             .unwrap(),
-            waiting_elements_subscriber: register_int_gauge_with_registry!(
-                "waiting_elements_subscriber",
-                "Number of waiting elements in the subscriber",
+            subscriber_download_payload_latency: register_histogram_with_registry!(
+                "subscriber_download_payload_latency",
+                "Time it takes to download a payload on the Subscriber",
+                // the buckets defined in seconds
+                vec![
+                    0.005, 0.01, 0.02, 0.05, 0.1, 0.25, 0.5, 1.0, 2.0, 3.0, 5.0, 10.0, 20.0, 40.0,
+                    60.0
+                ],
                 registry
             )
             .unwrap(),
+            subscriber_recovered_certificates_count: register_int_counter_with_registry!(
+                "subscriber_recovered_certificates_count",
+                "The number of certificates processed by Subscriber during the recovery period to fetch their payloads",
+                registry
+            ).unwrap(),
+            subscriber_download_payload_attempts: register_histogram_with_registry!(
```

Comment:
Nit: the name of the metric is `_retries` (i.e. attempts of order >1) and the documentation is attempts.